src/pages/dho/Members.vue

```diff
@@ -556,7 +557,7 @@ q-page.page-members
     )
     .cols.q-mt-md
       members-list(
-        :lastResult="this.$apollo.queries.daoApplicants.observer?.lastResult"
+        :lastResult="hasLastResult()"
```

Comment:
@arsenijesavic i can't make the computed property asynchronous because the result of the calculation depends on the result of the request to the server, so i use the method, let's leave it like this